docs/programatically-triggering-polling.adoc

```diff
@@ -20,3 +20,40 @@ With the background service disabled, trigger an update cycle by running the fol
 
     aktualizr --running-mode=once
 
+
+== Separately running individual update stages
+
+It is sometimes necessary to download an update when the is network connectivity then perform the installation later, without relying on the network. Note that aktualizr is always _safe_ against network drops. The worst that can happen if the network becomes unavailable is that the update will fail and need to be retried later.
```

Comment:
"when the is network connectivity" => "when there is network connectivity"

Reply:
Good catch. Fixed.